suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -314,7 +316,17 @@ export const startDescriptorPreloadedDiscoveryThunk = createThunk(
             return;
         }
 
-        const supportedNetworks = selectDiscoverySupportedNetworks(getState(), areTestnetsEnabled);
+        let supportedNetworks = selectDiscoverySupportedNetworks(getState(), areTestnetsEnabled);
+
+        // For development purposes, you can disable some networks to have quicker discovery in dev utils
+        if (isDebugEnv()) {
```

Comment:
Maybe this can be part of `discoveryConfigSlice.tsx` with all the other filtering (blacklisted, unavailable, ...).

Reply:
No, because that would affect the list I am using in dev utils to display all available options.